<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text File Extractor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Desktop/Extractor/FileExtractor/icon.ico</normaloff>../../../Desktop/Extractor/FileExtractor/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>100000</width>
     <height>100000</height>
    </size>
   </property>
   <widget class="QPushButton" name="btnBrowse">
    <property name="geometry">
     <rect>
      <x>636</x>
      <y>10</y>
      <width>36</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFileName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>495</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHeader">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>55</y>
      <width>146</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+++    USCDB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>76</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Header:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>55</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Footer:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFooter">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>55</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>---    END</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSplit">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>363</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Split</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Header Found:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHeaderFound">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>146</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 200);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFooterFound">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>146</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 200);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>135</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Footer Found:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkUnixEOL">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>105</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Unix/Linux Text File output</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>130</y>
      <width>276</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 200);</string>
    </property>
    <property name="text">
     <string>End of Line character is \r\n in windows and \n in Linux/Unix</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkIncludeHeaderFooter">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>180</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Include Header and Footer in output</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>373</y>
      <width>396</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>363</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxFileName">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>205</y>
      <width>661</width>
      <height>146</height>
     </rect>
    </property>
    <property name="title">
     <string>Filename</string>
    </property>
    <widget class="QRadioButton" name="radFileNameFixed">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>30</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Fixed filename template:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFileNameTemplate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>30</y>
       <width>116</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>_split%1.txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>346</x>
       <y>32</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Zero Pad:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFileNamePadding">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>421</x>
       <y>30</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radFileNameFromFile">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>77</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Filename from a line in the file.  Find:</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>186</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 200);</string>
     </property>
     <property name="text">
      <string>e.g.: _split00123.txt</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFileNameTextTemplate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>76</y>
       <width>116</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>IMSI = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>75</y>
       <width>186</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 200);</string>
     </property>
     <property name="text">
      <string>e.g.: If the line contains 'IMSI = ' use the rest of the line as the filename.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
